src/test_contracts/contracts/user_sample/Cargo.toml

```
@@ -0,0 +1,48 @@
+[package]
+name = "user_sample"
+version = "0.1.0"
+authors = ["sotnikov-s <sergei.s@p2p.org>"]
+edition = "2021"
+
+exclude = [
+  # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
+  "contract.wasm",
+  "hash.txt",
+]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[profile.release]
+opt-level = 3
+debug = false
+rpath = false
+lto = true
+debug-assertions = false
+codegen-units = 1
+panic = 'abort'
+incremental = false
+overflow-checks = true
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+library = []
+
+[dependencies]
+cosmwasm-std = { version = "1.0.0", features = ["staking"] }
+schemars = "0.8.10"
+serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+base64 = "0.13.0"
+
+# TODO: replace with sane imports once neutron-contracts repo sources are open
+interchain_queries = { path = "../../../../../neutron-contracts/packages/interchain_queries", default-features = false, version = "0.1.0"}
+neutron_sudo = { path = "../../../../../neutron-contracts/packages/neutron_sudo", default-features = false, version = "0.1.0"}
+neutron_bindings = {path = "../../../../../neutron-contracts/packages/bindings"}
```

The repo is already open, so:
```suggestion
interchain_queries = { git = "https://github.com/neutron-org/neutron-contracts", default-features = false, version = "0.1.0"}
neutron_sudo = { git = "https://github.com/neutron-org/neutron-contracts", default-features = false, version = "0.1.0"}
neutron_bindings = { git = "https://github.com/neutron-org/neutron-contracts" }
```

And you don't need an additional mount in the `build_release.sh` script

---

glad to know about it! done